pkg/httputils/http_utils.go

```diff
@@ -1,12 +1,99 @@
 package httputils
 
 import (
+	"context"
+	"fmt"
+	"io"
 	"net/http"
 	"strings"
+
+	"github.com/alibaba/pouch/apis/types"
```

Comment:
I do not think that a package in pkg like httputils should import packages from alibaba/pouch.  WDYT

Reply:
I think we can put it into the apis package level. It's only used in logs API.